src/test/fuzz/transaction.cpp

```diff
@@ -108,10 +108,10 @@ FUZZ_TARGET_INIT(transaction, initialize_transaction)
         }
     }
     if (!skip_tx_to_univ) {
-        TxToUniv(tx, /* hashBlock */ {}, u, true);
-        TxToUniv(tx, /* hashBlock */ {}, u, false);
+        TxToUniv(tx, /* hashBlock */ {}, /* include_addresses */ true, u);
+        TxToUniv(tx, /* hashBlock */ {}, false, u);
```

Comment:
nit: could mention `/* include_addresses */` here as well for symmetry

Reply:
squashed, and I included this nit in the push